test/mocks/plugins/MockAccessControlHookPlugin.sol

```diff
@@ -0,0 +1,80 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.25;
+
+import {PackedUserOperation} from "@eth-infinitism/account-abstraction/interfaces/PackedUserOperation.sol";
+
+import {PluginMetadata, PluginManifest} from "../../../src/interfaces/IPlugin.sol";
+import {IValidationHook} from "../../../src/interfaces/IValidationHook.sol";
+import {IStandardExecutor} from "../../../src/interfaces/IStandardExecutor.sol";
+import {BasePlugin} from "../../../src/plugins/BasePlugin.sol";
+
+// A pre validaiton hook plugin that uses per-hook data.
+// This example enforces that the target of an `execute` call must only be the previously specified address.
+// This is just a mock - it does not enforce this over `executeBatch` and other methods of making calls, and should
+// not be used in production..
+// solhint-disable custom-errors
```

Comment:
Nit: don't need this since we disabled it in config right?

Reply:
yep, removed